tests/test_validate.py

```diff
@@ -36,6 +36,7 @@ def test_validate_json_fail(self):
         In this particular test case, the 'mapping_justification' slot
         does not have EntityReference objects, but strings.
         """
+        # FIXME
```

Comment:
@hrshdhgd can you help me debug this?

Reply:
I was able to figure out what was going on. The test was incorrect. In order to avoid issues like this in the future, it would be good to make tests much more explicit (i.e., check that the error actually is what is expected)

Therefore, I'm skipping this test and adding a more detailed explanation as to why.

Reply:
New issue opened in https://github.com/mapping-commons/sssom-py/issues/434